hastory-server-data/src/Hastory/Server/Digest.hs

```diff
@@ -3,6 +3,7 @@
 
 module Hastory.Server.Digest where
 
+-- TODO: Move this under Server.Data
```

Comment:
Go for it.

Reply:
I decided against this for now. I might still do it, but I will wait until the rationale for it crops up again.